odl/solvers/nonsmooth/proximal_operators.py

```diff
@@ -24,12 +24,11 @@
 from __future__ import print_function, division, absolute_import
 import numpy as np
 
-from odl.operator import (
-    Operator, IdentityOperator, ScalingOperator, ConstantOperator,
-    DiagonalOperator, PointwiseNorm)
-from odl.set.space import LinearSpaceElement
+from odl.operator import (Operator, IdentityOperator, ScalingOperator,
+                          ConstantOperator, DiagonalOperator, PointwiseNorm,
+                          OperatorComp, MultiplyOperator)
 from odl.space import ProductSpace
-from odl.util import cache_arguments
+from odl.set.space import LinearSpaceElement
```

Comment:
This is all using the old master, you need to `git rebase master` to use the latest version

Reply:
What do you mean? I rebased on master and after removing this line, I get 29 pytest errors, e.g., NameError: name 'LinearSpaceElement' is not defined.

By the way, this line is not new in the code, see
https://github.com/odlgroup/odl/blob/master/odl/solvers/nonsmooth/proximal_operators.py

Reply:
`LinearSpaceElement` was replaced by `Tensor`, so the code needs to be updated wr.t. that.

Reply:
> LinearSpaceElement was replaced by Tensor, so the code needs to be updated wr.t. that.

Err, no. `FnBaseVector` was replaced by `Tensor`.

Reply:
Edit: it should be `SpaceElement`

Reply:
We didn't make that change yet, #1107 is still open.

Reply:
Nothing changed here.